packages/core/src/view/cell/VertexHandle.ts

```diff
@@ -129,6 +129,7 @@ class VertexHandle implements CellHandle {
         alpha2
       )
     );
+    this.setPosition(this.state.getPaintBounds(), pt, me);
```

Comment:
**praise:** Good catch. This seems to have been removed by mistake in a3102afa.
This line existed prior this commit and in the original mxGraph code

https://github.com/jgraph/mxgraph/blob/ff141aab158417bd866e2dfebd06c61d40773cd2/javascript/src/js/handler/mxHandle.js#L101-L105